backend/src/test/java/heartbeat/service/jira/JiraBoardConfigDTOFixture.java

```diff
@@ -494,38 +494,80 @@ public static StoryPointsAndCycleTimeRequest.StoryPointsAndCycleTimeRequestBuild
 			.treatFlagCardAsBlock(jiraBoardSetting.getTreatFlagCardAsBlock());
 	}
 
-	public static AllDoneCardsResponseDTO.AllDoneCardsResponseDTOBuilder ALL_DONE_CARDS_RESPONSE_FOR_ASSIGNEE_FILTER_METHOD_TEST() {
+	public static AllDoneCardsResponseDTO.AllDoneCardsResponseDTOBuilder ALL_DONE_CARDS_RESPONSE_FOR_ASSIGNEE_FILTER_TEST() {
 		return AllDoneCardsResponseDTO.builder()
-			.total("2")
+			.total("3")
 			.issues(List.of(
 					new JiraCard("ADM-475",
 							JiraCardField.builder()
 								.assignee(new Assignee("da pei"))
 								.status(new Status(CardStepsEnum.DONE.getValue()))
 								.build()),
+					new JiraCard("ADM-520",
+							JiraCardField.builder()
+								.assignee(null)
+								.status(new Status(CardStepsEnum.DONE.getValue()))
+								.build()),
 					new JiraCard("ADM-524",
 							JiraCardField.builder()
 								.assignee(new Assignee("xiao pei"))
 								.status(new Status(CardStepsEnum.DONE.getValue()))
 								.build())));
 	}
 
-	public static CardHistoryResponseDTO.CardHistoryResponseDTOBuilder CARD1_HISTORY_FOR_HISTORICAL_ASSIGNEE_FILTER_METHOD() {
+	public static CardHistoryResponseDTO.CardHistoryResponseDTOBuilder CARD1_HISTORY_FOR_HISTORICAL_ASSIGNEE_FILTER() {
+		return CardHistoryResponseDTO.builder()
+			.items(List.of(
+					new HistoryDetail(1673556350000L, "status", new Status(TESTING), new Status(REVIEW),
+							new HistoryDetail.Actor("da pei")),
```

Comment:
- what's the meaning of `"da pei"`
- duplicated code `"da pei"`, extract it to a fixture with meaningful varible name

Reply:
"da pei" is just a name represents the assignee or the Actor of jira card changes.
I have extracted this to a constant in our fixture.